src/components/layout/nav.js

```diff
@@ -1,1362 +0,0 @@
-// TODO: (discussion) make nav pure component, and move usage of nav to custom
```

Comment:
Why did you remove the whole `nav.js`?

Reply:
As Sean mentioned above, this file was removed from the upstream master branch, so I removed it as well.